https/README.md

```diff
@@ -15,17 +15,51 @@ The config file should be written in YAML format, and is reloaded on each connec
 
 ```
 tls_config:
```

Comment:
Thinking on it overnight, I think this should be tls_server_config. We already have a tls_config in multiple binaries for the client side.

Reply:
I was thinking the same. Updated.